ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -61,6 +61,14 @@ const addDividedLink = (links, name, href) => links.push((
   </span>
 ), <a key={name} href={href} target="_blank" rel="noreferrer">{name}</a>)
 
+const isTrnaOrRrna = (geneNames) => {
```

Comment:
you should be using the `gencodeGeneType` information from the gene to determine this, not the gene name

Reply:
I found that no mainTranscriptId was returned from the backend and the ES index looks normal. It should be a backend issue. But I have to stop here.

Reply:
I change to use the `gencodeGeneType` now. I couldn't see the field because my local reference data DB was bad. I updated the DB. It works now.